chassis.pp

```diff
@@ -1,3 +1,6 @@
+$memcache_config = sz_load_config()
```

Comment:
I'd be more inclined to put this inside `class memcache` in init.pp like I did over here: https://github.com/Chassis/Tester/blob/master/modules/tester/manifests/init.pp#L3